contents/docs/self-host/deploy/configuration.md

```diff
@@ -0,0 +1,199 @@
+    size: 10Gi
+  logRetentionBytes: _8_000_000_000
+
+# Extra replicas for more loaded services
+web:
+  replicacount: 2
+
+worker:
+  replicacount: 2
+
+plugins:
+  replicacount: 2
+```
+
+</details>
+
+<details>
+  <summary>
+    <b> Additional values to `values.yaml` for >1M events/month</b>
+  </summary>
+
+<br />
+
+```yaml
+# Note that this is experimental, please let us know how this worked for you.
+
+# More storage space
+clickhouseOperator:
+  storage: 200Gi
+
+postgresql:
+  persistence:
+    size: 60Gi
+
+redis:
+  master:
+    size: 30Gi
+
+kafka:
+  persistence:
+    size: 30Gi
+  logRetentionBytes: _22_000_000_000
+
+# Enable horizontal autoscaling for services
+pgbouncer:
+  hpa:
+    enabled: true
+
+web:
+  hpa:
+    enabled: true
+
+beat:
+  hpa:
+    enabled: true
+
+worker:
+  hpa:
+    enabled: true
+
+plugins:
+  hpa:
+    enabled: true
+```
+
+</details>
+
+### [ClickHouse](https://clickhouse.tech/)
+
+ClickHouse is the database that does the bulk of heavy lifting with regards to storing and analyzing the analytics data.
+
+By default, ClickHouse is installed as a part of the chart, powered by [clickhouse-operator](https://github.com/Altinity/clickhouse-operator/). As such it's important to set the database size to be enough to store the raw data via `clickhouseOperator.size` value.
+
+To use an external `ClickHouse` cluster, set `clickhouseOperator.enabled` to `false` and set `clickhouse.host`, `clickhouse.database`, `clickhouse.user` and `clickhouse.password`.
+
+_See [ALL_VALUES.md](ALL_VALUES.md) for full configuration options._
+
+### [PostgreSQL](https://www.postgresql.org/)
+
+> While ClickHouse powers the bulk of the analytics if you deploy PostHog using this chart, Postgres is still needed as a data store for PostHog to work.
+
+By default, PostgreSQL is installed as part of the chart. To use an external PostgreSQL server set `postgresql.enabled` to `false` and then set `postgresql.postgresHost` and `postgresql.postgresqlPassword`. The other options (`postgresql.postgresqlDatabase`, `postgresql.postgresqlUsername` and `postgresql.postgresqlPort`) may also want changing from their default values.
+
+To avoid issues when upgrading this chart, provide `postgresql.postgresqlPassword` for subsequent upgrades. This is due to an issue in the PostgreSQL chart where password will be overwritten with randomly generated passwords otherwise. See https://github.com/helm/charts/tree/master/stable/postgresql#upgrade for more detail.
+
+_See [ALL_VALUES.md](ALL_VALUES.md) and [PostgreSQL chart](https://github.com/bitnami/charts/tree/master/bitnami/postgresql) for full configuration options._
+
+### [Redis](https://redis.io/)
+
+By default, Redis is installed as part of the chart. To use an external Redis server/cluster set `redis.enabled` to `false` and then set `redis.host`. If your redis cluster uses password define it with `redis.password`, otherwise just omit it. Check the table above for more configuration options.
```

Comment:
```suggestion
By default, Redis is installed as part of the chart. To use an external Redis server/cluster set `redis.enabled` to `false` and then set `redis.host`. If your Redis cluster uses a password define it with `redis.password`, otherwise just omit it.
```